src/Symfony/Component/Validator/Tests/Constraints/LengthValidatorTest.php

```diff
@@ -89,12 +89,11 @@ public function getFiveOrMoreCharacters()
         );
     }
 
+    /**
+     * @requires extension mbstring
```

Comment:
e don't care about tests portability, the only that counts are the continuously tested ones